src/SamplesApp/SamplesApp.UITests/SamplesApp.UITests.csproj

```diff
@@ -9,7 +9,7 @@
 		<PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.10.0" />
 		<PackageReference Include="NUnit" />
 		<PackageReference Include="NUnit3TestAdapter" /> 
-		<PackageReference Include="Xamarin.UITest" Version="3.0.15" />
+		<PackageReference Include="Xamarin.UITest" Version="3.0.19" />
```

Comment:
I think this should be specified in `Directory.Build.targets` instead...  @jeromelaban?

Reply:
We can yes, but don't update to 3.0.19, it does not work properly for android.